### PR TITLE
Header/body does not exist anymore. Deprecate and remove own internal…

### PR DESCRIPTION
**Pull request by baldersheim**
… usage.

@arnej27959 PR

### CODE REVIEW

**Comment by baldersheim** [comment]
@arnej27959 Ping

File: document/src/tests/documenttestcase.cpp [hunk 1]
```diff
@@ -585,7 +585,7 @@ TEST(DocumentTest, testReadSerializedFile)
 
     EXPECT_TRUE(buf2.empty());
     buf2.rp(0);
-    EXPECT_EQ(len - 13, buf2.size()); // Size is smaller as we are merging to one chunk.
+    EXPECT_EQ(len, buf2.size());
```
Comment (vekterli) [inline document/src/tests/documenttestcase.cpp]: Do we have any files checked in testing that we can read from legacy multi-chunk serializations, or is this sufficiently ancient by now that we don't consider it necessary anymore?

**Review by vekterli — APPROVED** [review]
:+1:

File: document/src/tests/documenttestcase.cpp (hunk 1 above)
Comment (baldersheim) [reply document/src/tests/documenttestcase.cpp]: It is fairly ancient, but I still think we have some.
I will check.

**Pull request merged**